themes/default/layouts/page/about.html

```diff
@@ -1,25 +1,173 @@
 {{ define "hero" }}
-    {{ partial "hero" (dict "title" "About Us") }}
+    {{ partial "hero" (dict "title" "Program the Cloud") }}
 {{ end }}
 
 {{ define "main" }}
     <section class="max-w-4xl container mx-auto py-8 px-4 text-center text-lg">
         <p>
-            Pulumi is a venture backed startup in Seattle whose mission is to enable every
-            person to harness the power of the cloud. Our team is a diverse and talented group from all
-            corners of our industry, and our products sit uniquely at the intersection of
-            developer tools and cloud operations.
-        </p>
-        <p>
-            Get your code to the cloud. Faster. Together.
+            We help developers and infrastructure teams work better together to build
+            modern cloud software.
         </p>
         <p class="text-3xl">
             <a href="{{ relref . "/careers" }}">We are hiring!</a>
         </p>
     </section>
 
+    <section id="values" class="px-4 pb-8">
+        <div class="container mx-auto">
+            <h2 class="mb-8 text-center">Our Values</h2>
+            <div class="lg:flex ml-16 mr-16">
+                <div class="lg:w-1/4 p-5">
+                    <div class="card bg-white max-w-md mx-auto">
+                        <div class="px-8 py-8">
+                            <div class="icon-section pb-4">
+                                <i class="fas fa-users text-violet-600 text-3xl"></i>
+                            </div>
+                            <div>
+                                <h6>Work as a Team</h6>
+                            </div>
+                            <div class="description">
+                                <p class="text-sm">
+                                    We believe that working together is critical to our success
```

Comment:
i think this should be rephrased into what we do instead of what we believe. something like "we work together..."